src/debug_menu.cpp

```diff
@@ -329,6 +330,7 @@ static int map_uilist()
         { uilist_entry( debug_menu_index::CHANGE_WEATHER, true, 'w', _( "Change weather" ) ) },
         { uilist_entry( debug_menu_index::WIND_DIRECTION, true, 'd', _( "Change wind direction" ) ) },
         { uilist_entry( debug_menu_index::WIND_SPEED, true, 's', _( "Change wind speed" ) ) },
+        { uilist_entry( debug_menu_index::GEN_SOUND, true, 'G', _( "Generate sound" ) ) },
```

Comment:
Maybe `S` hotkey fits here better?

Reply:
It does... I didn't put much thought into the letter.